app/views/skills/show.html.haml

```diff
@@ -1,7 +1,32 @@
+= render "remote_modal", title: "Skill: #{@skill.title} (#{@skill.people_skills.where.not(level: nil, interest: nil).count} Members)" do
+  %div.d-flex.flex-column
+    - @skill.people_skills.each do |person_skill|
+      - if person_skill.level
+        %div.d-flex.flex-row.align-items-center.skill-row
+          %div.col-2
+            = link_to Person.find(person_skill.person_id).name, person_path(person_skill.person_id)
+          %div.col-2.d-flex.justify-content-center
+            %p.m-0= @skill.title
+          %div.col-2
+            %div.d-flex.flex-column
+              %label.form-label.text-gray= t ExpertiseTopicSkillValue.skill_levels.key(person_skill.level - 1)
```

Comment:
General thing for this whole file:
From what we learned above `level: 1` and `interest: 1` are the lowest possible value that can be selected. Therefore a people_skill with  `level: 1` and `interest: 1` should display both bubbles completely to the left which currently is not the case.